frontend/src/static/js/main.js

```diff
@@ -250,7 +250,11 @@ window.app = new window.Vue({
       return JSON.stringify(dataObj);
     },
 
-    getRepoSenseLink() {
+    getRepoSenseHomeLink() {
+      return 'https://reposense.org';
```

Comment:
It seems Chrome `https://reposense.org` causes browsers to give a security warning but `reposense.org` does not. Is it the same for you guys?

Reply:
Just checked. This error didn't come for me yesterday. But it is coming right now.

Reply:
Should we omit `https://` part?

Reply:
Seems to be fine now.

@damithc 
If its convenient for you, adding a `CNAME` record to `https://reposense.org` will be the best course of action as the url shorter service doesn't seem to be reliable.

Reply:
Yup. Seems to be okay now.

Reply:
I'm still getting the error. 
GoDaddy (our domain vendor) is supposed to redirect to GitHub.
Is there any downside to omitting https:// part? The browser seems to go to https:// anyway.

Reply:
i believe the problem is that the redirection from `reposense.org` to GitHub is just a service provided by the domain provider. Using `https://` in this case is incorrect as `reposense.org` is not tied to any physical server with ssl cert. 

Alternative, if we really want to add the `https://`, we can consider creating a gh page for reposense. GitHub provides the service where you can use your own domain name for your gh page, as well as the ssl certificate when you do so.